app/env.ts

```diff
@@ -3,12 +3,15 @@ import { defineConfig, Schema } from '@julr/vite-plugin-validate-env';
 export default defineConfig({
     APP_TITLE: Schema.string(),
     APP_ENVIRONMENT: (key, value) => {
-        const regex = /^production|staging|testing|alpha-\d+|development$/;
+        const regex = /^production|staging|testing|alpha-\d+|development|APP_ENVIRONMENT_PLACEHOLDER$/;
```

Comment:
We should add a NOTE on when `APP_ENVIRONMENT_PLACEHOLDER` is used.

Reply:
There is a console message below that.